grammar.js

```diff
@@ -672,12 +674,16 @@ module.exports = grammar({
       '}'
     ),
 
+    parenthesized_expression: $ => prec(PREC.parenthesized_expression, seq(
```

Comment:
This precedent is required to disambiguate between `parenthesized_expression` and `tuple`. `parenthesized_expression` is used in `yield`, whose production consists of either `_expression` or `expression_list`.